media-video/droidcam/droidcam-1.4.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1 desktop linux-mod xdg-utils
```

Comment:
The xdg eclass (instead of xdg-utils) might save you 4 lines, it condenses updating desktop, mimeinfo and icon cache into a single line

Reply:
Moved from xdg-utils to xdg.